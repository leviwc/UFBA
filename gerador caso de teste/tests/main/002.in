22 217
17 1
15 22
16 13
13 13
16 21
11 4
2 13
7 3
17 5
9 14
11 14
8 20
5 5
21 9
16 2
18 2
3 2
7 14
3 21
20 14
1 1
22 22
15 6
22 3
11 16
12 4
3 17
3 19
8 7
1 20
7 16
15 18
5 14
4 9
12 16
2 14
6 21
4 8
21 10
12 6
17 14
19 6
4 7
17 11
9 5
21 12
17 15
1 9
2 12
14 12
6 13
21 13
21 8
4 16
11 12
16 9
22 5
1 19
1 2
18 16
13 18
20 19
9 17
7 7
7 13
8 9
7 19
21 18
21 5
12 12
19 13
2 20
6 8
1 4
3 11
19 12
18 3
6 20
13 5
10 3
18 20
5 7
20 17
18 11
19 14
15 12
22 18
4 22
19 8
7 11
14 16
10 12
9 19
12 8
15 21
4 2
3 1
18 12
22 21
12 9
6 9
6 16
1 13
6 14
8 10
9 9
7 21
9 7
9 11
20 13
9 2
2 2
4 17
5 15
2 21
18 7
10 17
1 10
19 17
6 10
10 7
6 7
11 8
17 13
5 16
15 11
8 3
18 18
19 18
18 10
5 4
9 20
21 4
20 15
8 2
1 8
6 18
2 22
20 5
10 4
6 17
16 10
17 2
11 6
13 9
2 11
15 8
5 2
14 4
18 9
12 3
8 5
14 22
1 18
15 3
15 7
15 15
19 5
17 7
3 5
10 10
22 7
6 1
20 3
19 4
22 11
3 13
15 10
5 10
11 10
8 22
2 7
16 8
19 16
20 10
17 12
21 20
3 16
19 10
1 5
21 11
22 9
22 20
20 20
19 22
1 16
4 18
3 3
6 6
9 15
20 12
6 4
1 21
21 19
14 15
12 5
1 11
18 5
10 22
3 4
10 2
16 22
6 3
19 11
22 6
13 14
17 16
13 22
16 16
7 20
15 1
10 13
8 17
14 21
9 10
15 13
3 14
