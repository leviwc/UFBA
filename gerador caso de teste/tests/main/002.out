Resultado com busca binaria: 3
36.053000 s
Resultado com backtracking apenas: 3
0.016000 s
Resultado testando todas as possibilidades: 3
121.619000 s
Resultado com busca binaria otimizada: 3
7.760000 s
